app/controllers/admin/rents_controller.rb

```diff
@@ -35,7 +35,7 @@ def update
   end
 
   def destroy
-    @rent.destroy
+    @rent.destroy!
```

Comment:
Now you see why using the ! method is important :) it's so easy to miss and then you have a silent error.